Core/HLE/proAdhoc.h

```diff
@@ -50,7 +50,7 @@ class PointerWrap;
 #include <fcntl.h>
 #include <errno.h>
 #endif
-#ifdef _MSC_VER
+#ifdef _WIN32
 #define PACK
```

Comment:
Hmm, I would think PACK would be _MSC_VER related.  The others look win32 related.

I think `#define PACK __attribute__((packed))` would be needed when _WIN32 but not _MSC_VER right?

-[Unknown]